src/modules/queue/components/Queue.jsx

```diff
@@ -36,11 +33,10 @@ export default function Queue({isQueueOpen, isSearchOpen, queue, toggleQueue, to
 }
 
 const QueueList = styled.div`
-	background: white;
-	min-height: 37.5rem;
 	max-width: 56.25rem;
-	margin: auto;
+	margin: 3.125rem auto 0 auto;
```

Comment:
If you remove `position: fixed` on `InfoHeader` you don't need this either.

Reply:
The reason I'm calling the InfoHeader fixed, is because I want it to stay put rather than scrolling away when there are more than 10 songs in the list. This is still broken but I can work to fix it

![position-fixed scroll](https://user-images.githubusercontent.com/7230519/45183517-5620eb00-b1f2-11e8-9b7e-5adae53fd92f.gif)